analyses/tp53_nf1_score/05-tp53-altered-annotation.Rmd

```diff
@@ -408,22 +432,25 @@ polya_tp53<- as.data.frame(subset_polya) %>%
 Plot distribution of TP53 gene expression (poly-a)
 
 ```{r, out.width="50%" }
-
-ggplot(polya_tp53, aes(x = factor(tp53_altered), y = TP53)) +
-  geom_violin()+
-  geom_jitter(alpha = 0.5, width = 0.2) +
-  stat_compare_means() +
-  theme_bw() +
-  ggtitle("Distribution of TP53 expression across tp53 altered status") +
-  theme(axis.text.x = element_text(angle = 60, hjust = 1))+
-  xlab("tp53 altered status") 
+ggviolin(polya_tp53, x = "tp53_altered", y = "TP53", 
+         color = "tp53_altered", 
+         palette = "jco",
+         order = c("activated", "loss"),
+         add = "boxplot",  
+   ggtheme = theme_pubr()) +
+  # Add pairwise comparisons p-value
+  stat_compare_means(method = "wilcox.test", label.y = 80) +
+  xlab("TP53 altered status") +
+  ylab("TP53 RNA expression") +
+  rremove("legend")
+
+ggsave(paste0(plot_dir, "/tp53_expression_by_altered_status_polya.png"))
```

Comment:
>I think we need a little more information about how stat_compare_means() works in these comments. When there are multiple comparisons happening, does any correction get applied?

Thanks - I can either update the README with a blurb or add the [link](http://www.sthda.com/english/articles/24-ggpubr-publication-ready-plots/76-add-p-values-and-significance-levels-to-ggplots/#compare_means) and an explanation within the RMD file.  Actually, let me add the calculations to print in the notebook as well - there is a correction in `compare_means` when >2 groups, and there is an option to add the adj p value on the plot instead of the first test pval, so I can update that as well. Thanks for bringing that up.

>file.path() is probably a little more "standard" over using paste0() with the system file pathway separator hardcoded.

I was thinking this looked off, but it was late. Thanks!